tests/js/spec/views/projectReleasesLanding.spec.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+
+import ReleaseLanding from 'app/views/projectReleases/releaseLanding';
+import ReleaseProgress from 'app/views/projectReleases/releaseProgress';
+
+describe('ReleaseLanding', function() {
+  describe('renders and progresses', function() {
+    it('should show first card', function() {
+      let routerContext = TestStubs.routerContext();
+      let wrapper = mount(<ReleaseLanding />, routerContext);
+      expect(wrapper).toMatchSnapshot();
```

Comment:
We probably don't need the snapshot because the below expectations tell us exactly what you're testing.